### PR TITLE
ServiceBus: 2018-01-01 Preview - for managing Namespace, IPFilter Rules, VirtualNetworkRules and Zone Redundant

### PR DESCRIPTION
**Pull request by v-Ajnava**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json [hunk 1]
```diff
@@ -0,0 +1,1175 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ServiceBus2018PreviewManagementClient",
```
[the remark this answers is not in the document]
Comment (v-Ajnava) [reply specification/servicebus/resource-manager/Microsoft.ServiceBus/preview/2018-01-01-preview/servicebus-preview.json]: resolved, reverted the change

File: specification/servicebus/resource-manager/readme.md [hunk 2]
```diff
@@ -54,6 +54,14 @@ input-file:
 - Microsoft.ServiceBus/stable/2015-08-01/servicebus.json
 ```
 
+### Tag: package-2018-01-preview
```
Comment (lmazuel) [inline specification/servicebus/resource-manager/readme.md]: Change too the default tag at the beginning of the file.

Comment (v-Ajnava) [reply specification/servicebus/resource-manager/readme.md]: resolved, move to top.

File: specification/servicebus/resource-manager/readme.md [hunk 3]
```diff
@@ -29,6 +29,14 @@ openapi-type: arm
 tag: package-2017-04
 ```
 
+### Tag: package-2018-01-preview
```
Comment (lmazuel) [inline specification/servicebus/resource-manager/readme.md]: You need to change line 29 (the default tag)

**Review by lmazuel — APPROVED** [review]
Preview, content seems ok. Still wait ARM feedback.

**Comment by ravbhatnagar** [comment]
@v-Ajnava - Looks like you are adding new resource types. Have these ever been reviewed by ARM team in person? If not, please set up a call with armapireview to review these. @simongdavies FYI

**Comment by v-Ajnava** [comment]
@ravbhatnagar sent a meeting invite to armapireview

**Comment by v-Ajnava** [comment]
Fixed the camel case and description for subnet id noticed during ARM review meeting.

**Comment by v-Ajnava** [comment]
@ravbhatnagar i have implemented the ARM review meeting change, can you please review the PR?

**Comment by ravbhatnagar** [comment]
Signing off frm ARM.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**